### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

This one fell through the cracks 🤦‍♂️ . Adding docs.

For https://github.com/elastic/ecs-logging-go-zerolog/issues/2.

### CODE REVIEW

File: docs/setup.asciidoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+[float]
+[[setup-step-2]]
+=== Step 2: Configure
+
+Set up a default logger. For example:
+
+[source,go]
+----
+logger := ecszerolog.New(os.Stdout)
+log.Logger = logger
```
Comment (bmorelli25) [inline docs/setup.asciidoc:24]: From readme

File: docs/setup.asciidoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+[source,json]
+----
+{
+  "@timestamp": "2021-01-20T11:12:43.061+0800",
+  "custom":"foo",
+  "ecs.version": "1.6.0",
+  "log.level": "info",
+  "message":"hello world"
+}
```
Comment (bmorelli25) [inline docs/setup.asciidoc:51]: My best attempt

**Review by michel-laterman — APPROVED** [review]
lgtm
thanksfor getting this done!

**Comment by bmorelli25** [comment]
Thanks, Michel. Do you know happen to know who has merge permissions on this repo?

**Comment by bmorelli25** [comment]
👀 it looks like @axw might...

**Review by axw — APPROVED** [review]

**Pull request merged**